frontend/src/components/object/ObjectVersion.vue

```diff
@@ -39,15 +40,18 @@ const { getUserSearch } = storeToRefs(userStore);
 const tableData: Ref<Array<VersionDataSource>> = ref([]);
 
 // Actions
+const router = useRouter();
 const toast = useToast();
 
-// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-const showInfo = async (id: string) => {
-  // emit('show-object-info', id);
-};
-
-function onDeletedSuccess() {
+async function onDeletedSuccess() {
```

Comment:
maybe passing objectId and versionId as params of this function would help.
not sure the pattern for emitting that